src/__configuration__/themes/thanksgiving.ts

```diff
@@ -0,0 +1,68 @@
+import * as PXBThemes from '@pxblue/react-themes';
+import * as Colors from '@pxblue/colors';
+import * as BrandingColors from '@pxblue/colors-branding';
+import Thanksgiving from '../../app/assets/themes/thanksgiving-banner.jpg';
+import turkeyCursor from '../../app/assets/themes/turkey-cursor.png';
+import AppBarTile from '../../app/assets/themes/thanksgiving-appbar-tile.png';
+import { Schedule } from './types';
+
+export const ThanksgivingSchedule: Schedule = {
+    start: new Date(0, 10, 22), // Nov 22
+    end: new Date(0, 10, 29), // Nov 29
```

Comment:
This will need to be December 1 in order to run _through_ Nov 30
```suggestion
    end: new Date(0, 11, 1), // Nov 30
```